detect_secrets/filters/heuristic.py

```diff
@@ -165,3 +165,12 @@ def is_indirect_reference(secret: str) -> bool:
             continue
 
     return output
+
+
+def is_lock_file(filename: str) -> bool:
+    return os.path.basename(filename) in {
+        'Brewfile.lock.json',
+        'composer.lock',
+        'package.lock',
+        'yarn.lock',
```

Comment:
I've also seen `Podfile.lock`, `Cartfile.resolved`, and `Package.resolved`

is `package.lock` a typo for `package-lock.json`?

Reply:
TIL about those other entries. I've updated this, and added tests too.